mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -86,8 +88,11 @@ CLASS net/minecraft/class_31 net/minecraft/item/ItemStack
 	METHOD method_717 isDamageable ()Z
 	METHOD method_718 equals2 (Lnet/minecraft/class_31;)Z
 		ARG 1 stack
+	METHOD method_719 hasSubtypes ()Z
 	METHOD method_720 isDamaged ()Z
+	METHOD method_721 getDamage2 ()I
```

Comment:
Same issue as previous comment about the "equals2" method, name is not descriptive enough.

Reply:
They appaer to be doing the exactly same thing
![obrazek](https://github.com/calmilamsy/biny-mappings/assets/24621316/9536d4ba-127d-4f03-a46d-22d4b8bb3b22)

Reply:
In such cases I'd say that it's best to leave the one with least calls unmapped, but suffixing with "2" is also fine with me.